### DISCUSSION TITLE
How to set acceptable double click delay "range"

### DISCUSSION THREAD

**Discussion by zjwhitehead** [bxparks/AceButton#88] [Q&A] [answered]
I have the double click event working but I cant find how to configure the upper and lower bounds for whats an acceptable delay between clicks
I know there is some debouncing and the lower end default is pretty good but how do allow, for example, up to 1000ms between clicks for it to count as a double click?

Thanks and love the library!

**Comment by bxparks** [comment, marked as answer]
I think you are looking for `setDoubleClickDelay()` as described in this section:
https://github.com/bxparks/AceButton#TimingParameters
